cmd/exporters/influxdb/influxdb.go

```diff
@@ -121,7 +125,7 @@ func (e *InfluxDB) Init() error {
 	// construct HTTP client
 	e.client = &http.Client{Timeout: timeout}
 
-	e.Logger.Debug().Msgf("initialized exporter, ready to emit to [%s:%s]", addr, port)
+	//e.Logger.Debug().Msgf("initialized exporter, ready to emit to [%s:%s]", *addr, *port)
```

Comment:
why comment out?

Reply:
mentioned above due to NPE in one of the test cases when url is not null.

Reply:
handled this.

Reply:
handled this.